packages/fury-adapter-oas3-parser/test/unit/parser/parseYAML-test.js

```diff
@@ -318,54 +378,75 @@ describe('#parseYAML', () => {
     expect(warning).to.have.sourceMapEndColumn(13);
   });
 
-  it('can parse a set into an array element', () => {
-    const element = parseYAML('!!set\n  - one\n  - two', context);
-
-    expect(element).to.be.instanceof(namespace.elements.ParseResult);
-    expect(element.length).to.equal(2);
-
-    const arry = element.first;
-
-    expect(arry).to.be.instanceof(namespace.elements.Array);
-    expect(arry.toValue()).to.deep.equal(['one', 'two']);
-    expect(arry).to.have.sourceMapStart(0);
-    expect(arry).to.have.sourceMapOffset(21);
-    expect(arry).to.have.sourceMapStartLine(1);
-    expect(arry).to.have.sourceMapStartColumn(1);
-    expect(arry).to.have.sourceMapEndLine(3);
-    expect(arry).to.have.sourceMapEndColumn(8);
-
-    const entryOne = arry.get(0);
-
-    expect(entryOne).to.be.instanceof(namespace.elements.String);
-    expect(entryOne.toValue()).to.equal('one');
-    expect(entryOne).to.have.sourceMapStart(10);
-    expect(entryOne).to.have.sourceMapOffset(3);
-    expect(entryOne).to.have.sourceMapStartLine(2);
-    expect(entryOne).to.have.sourceMapStartColumn(5);
-    expect(entryOne).to.have.sourceMapEndLine(2);
-    expect(entryOne).to.have.sourceMapEndColumn(8);
-
-    const entryTwo = arry.get(1);
-
-    expect(entryTwo).to.be.instanceof(namespace.elements.String);
-    expect(entryTwo.toValue()).to.equal('two');
-    expect(entryTwo).to.have.sourceMapStart(18);
-    expect(entryTwo).to.have.sourceMapOffset(3);
-    expect(entryTwo).to.have.sourceMapStartLine(3);
-    expect(entryTwo).to.have.sourceMapStartColumn(5);
-    expect(entryTwo).to.have.sourceMapEndLine(3);
-    expect(entryTwo).to.have.sourceMapEndColumn(8);
-
-    const warning = element.second;
-    expect(warning).to.be.instanceof(namespace.elements.Annotation);
-    expect(warning.toValue()).to.equal('Interpreting YAML !!set as array');
-    expect(warning).to.have.sourceMapStart(0);
-    expect(warning).to.have.sourceMapOffset(21);
-    expect(warning).to.have.sourceMapStartLine(1);
-    expect(warning).to.have.sourceMapStartColumn(1);
-    expect(warning).to.have.sourceMapEndLine(3);
-    expect(warning).to.have.sourceMapEndColumn(8);
+  describe('set', () => {
+    it('can parse a set into an array element', () => {
+      const element = parseYAML('!!set\n  - one\n  - two', context);
+
+      expect(element).to.be.instanceof(namespace.elements.ParseResult);
+      expect(element.length).to.equal(2);
+
+      const arry = element.first;
```

Comment:
lowercased `array` isn't a keyword in JavaScript

Reply:
or is it a typo?

Reply:
This was the code before that I just indented, I'll change the occurance of the word